src/terraform/commands.go

```diff
@@ -3,14 +3,30 @@ package terraform
 import (
 	"fmt"
 
+	"github.com/Originate/exosphere/src/docker/tools"
 	"github.com/Originate/exosphere/src/types"
 	"github.com/Originate/exosphere/src/util"
 )
 
 // RunInit runs the 'terraform init' command and force copies the remote state
 func RunInit(deployConfig types.DeployConfig) error {
+	homeDir, err := util.GetHomeDirectory()
+	if err != nil {
+		return err
+	}
+	volumes := []string{
+		fmt.Sprintf("%s:/app", deployConfig.TerraformDir),
+		fmt.Sprintf("%s/.aws:/root/.aws", homeDir),
+	}
```

Comment:
I think this should be inline since its small enough. Gets rid of an extra variable. Same for below